sklearn/manifold/_mds.py

```diff
@@ -275,14 +289,18 @@ def smacof(
         Whether or not to return the number of iterations.
 
     normalized_stress : bool or "auto" default="auto"
-        Whether use and return normed stress value (Stress-1) instead of raw
-        stress calculated by default. Only supported in non-metric MDS.
+        Whether use and return normalized stress value (Stress-1) instead of raw
+        stress. By default, metric MDS uses raw stress while non-metric MDS uses
+        normalized stress.
```

Comment:
```suggestion
        Whether to return normalized stress value (Stress-1) instead of raw
        stress. By default, metric MDS returns raw stress while non-metric MDS returns
        normalized stress.
```

Reply:
Agree, removed "use", as you suggested.